cumulusci/utils/metadeploy.py

```diff
@@ -0,0 +1,24 @@
+"""Utilities for working with MetaDeploy objects"""
+
+from cumulusci.core.config import FlowConfig, TaskConfig
+from cumulusci.core.flowrunner import FlowCoordinator
+from cumulusci.utils import cd
+
+
+def get_frozen_steps(project_config, plan_config):
+    """Return a list of frozen steps for a MetaDeploy plan config"""
+    flow_config = FlowConfig(plan_config)
+    flow_config.project_config = project_config
+    flow = FlowCoordinator(project_config, flow_config)
+    steps = []
+    for step in flow.steps:
+        if step.skip:
+            continue
+        with cd(step.project_config.repo_root):
+            task = step.task_class(
+                step.project_config,
+                TaskConfig(step.task_config),
+                name=step.task_name,
+            )
+            steps.extend(task.freeze(step))
+    return steps
```

Comment:
@boakley @jstvz To avoid circular import problems, cumulusci/utils is designated for utility modules that stand alone and do not have dependencies on other parts of CumulusCI. This module would belong better in cumlusci/core somewhere.

Reply:
@boakley I think it makes sense to stay as a new, separate module, cumulusci/core/metadeploy.py. It's a different domain of functionality than the existing functions in cumulusci/core/utils.py